recipes/autoconf/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import os
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+
+
+class AutoconfConan(ConanFile):
+    name = "autoconf"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/autoconf/"
+    description = "Autoconf is an extensible package of M4 macros that produce shell scripts to automatically configure software source code packages"
+    topics = ("conan", "autoconf", "configure", "build")
+    license = ("GPL-2.0-or-later", "GPL-3.0-or-later")
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+
+    def build_requirements(self):
+        if tools.os_info.is_windows and "CONAN_BASH_PATH" not in os.environ:
+            self.build_requires("msys2/20190524")
+
+    def _configure_autotools(self):
+        autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
```

Comment:
Usually, the AutoTollsBuildEnvironment is pinned in an  `_autotools`  attribute, to avoid calling configure twice (eg https://github.com/conan-io/conan-center-index/blob/24311c41e61a428eb20c6b715e14ce6e420fbbe9/recipes/libuuid/all/conanfile.py#L21). Is there a reason not to do it here ?

Reply:
No reason.
I didn't realize there was a tradition to do do so.